src/frontend/citizen-portal/nginx.conf

```diff
@@ -14,3 +22,7 @@ error_page   500 502 503 504  /50x.html;
 location = /50x.html {
   root   /usr/share/nginx/html;
 }
+
+location /api/v1/ {
```

Comment:
Did you verify this is correct?  I didn't expect `/v1/`

Reply:
we could remove it, no big deal.
Make it just /api